src/System.Net.Security/tests/FunctionalTests/ClientDefaultEncryptionTest.cs

```diff
@@ -43,7 +43,7 @@ public ClientDefaultEncryptionTest()
 
                 using (var sslStream = new SslStream(client.GetStream(), false, AllowAnyServerCertificate, null))
```

Comment:
Please ensure that all modified tests are enabled. (I disabled some that were failing intermittently in CI.)